addons/web/static/tests/utils.js

```diff
@@ -0,0 +1,550 @@
+/* @odoo-module */
+
+import { registerCleanup } from "@web/../tests/helpers/cleanup";
```

Comment:
The goal is to quickly move forward, I don't want to preemptively remove the legacy file without having the new utils available to help me doing so.

If it's really an issue of dependency I can just copy/paste the content of the legacy file into the new one, but the end result in a few PR down the road will be the same.

About not putting it in `helpers/utils`, it is because I defined a click method (and a few others), but there is already another version of these methods in that file, and they are incompatible.
The currently existing methods attempt to do their action immediately (and crash if not possible), and then they wait a little bit of time before resolving.
The new methods also attempt to do their action immediately, but if it fails they will wait before the conditions are met, and then do their action, with no waiting time afterwards.

The are incompatible because the new methods requires the tests to be adapted so that each step await for what they need, rather than hope that previous steps waiting an arbitrary time for their state to be complete.